SpatialGDK/Source/SpatialGDK/Public/Schema/Ping.h

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Schema/Component.h"
+#include "SpatialConstants.h"
+#include "Utils/SchemaUtils.h"
+
+#include <WorkerSDK/improbable/c_schema.h>
+#include <WorkerSDK/improbable/c_worker.h>
+
+namespace improbable
+{
+
+	struct ServerPing : Component
+	{
+		static const Worker_ComponentId ComponentId = SpatialConstants::SERVER_PING_COMPONENT_ID;
+
+		ServerPing() = default;
+		ServerPing(const Worker_ComponentData& Data)
+		{
+		}
+
+		FORCEINLINE Worker_ComponentData CreateServerPingData()
```

Comment:
Why not just make this a non member function, or a static function? Is there a convention that we have for defining such components?

Reply:
Yep, this is our convention, so schema data for any hand-crafted component can be created like
```
Components.Add(improbable::MySchema(Arguments...).CreateMySchemaData());
```

Reply:
Ah, thank you. I'll resolve this then.